uaclient/config.py

```diff
@@ -248,7 +248,16 @@ def contract_remaining_days(self) -> int:
     @property
     def features(self):
         """Return a dictionary of any features provided in uaclient.conf."""
-        return self.cfg.get("features", {})
+        features = self.cfg.get("features")
+        if features:
+            if isinstance(features, dict):
+                return features
+            else:
+                logging.warning(
+                    "Unexpected uaclient.conf features value. Expected dict, but found %s",
+                    features,
```

Comment:
should we have here `str(features)`  instead ?

Reply:
I thought %s formatting does that for us right?

Reply:
Yep, it does. My mistake here